src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetNativeVariantForObjectTests.cs

```diff
@@ -119,10 +119,6 @@ public static IEnumerable<object[]> GetNativeVariantForObject_NonRoundtrippingPr
             {
```

Comment:
These `!PlatformDetection.IsNetCore` checks can be delete too.

Reply:
And tests moved to the methods later in the file that validate the failure?

Reply:
> And tests moved to the methods later in the file that validate the failure?

I don't quite get what this means.

Reply:
I meant that we should preserve the test cases being removed, but change the expected behavior for them.

I assume that the expected behavior for these test cases is some kind of exception. There are theories later in this file that are testing different types of exceptions, so we may be able to just move the the cases to there theories.

(Ignore this if we have the appropriate coverage for these cases already somewhere.0